tool/tsh/db.go

```diff
@@ -62,10 +107,112 @@ func onListDatabases(cf *CLIConf) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	showDatabases(cf.SiteName, databases, activeDatabases, cf.Verbose)
+
+	showDatabases(cf.SiteName, displayUsersForDb, databases, activeDatabases, cf.Verbose)
 	return nil
 }
 
+// getUsersForDb finds allowed and denied users given user's effective roleSet and database in consideration. The result is a user-readable string.
+func getUsersForDb(roleSet services.RoleSet, database types.Database) string {
+	if len(roleSet) == 0 {
+		return ""
+	}
+
+	allowedUsers, deniedUsers := listAllowedDeniedDbUsers(roleSet)
+
+	allUsers := make([]string, 0)
+	allUsers = append(allUsers, allowedUsers...)
+	allUsers = append(allUsers, deniedUsers...)
```

Comment:
```suggestion
	allUsers := append(allowedUsers, deniedUsers...)
```

Also, it feels like the above function can just return all users right away since allowed/denied aren't used independently here.